digitransit-component/packages/digitransit-component-datetimepicker/src/helpers/Datetimepicker.js

```diff
@@ -240,6 +251,74 @@ function Datetimepicker({
     }
   }
 
+  const validateInputTime = inputValue => {
+    const validateClock = (hours, minutes) => {
```

Comment:
Input validation is now confusing, for many reasons:
1. There are two validation functions. Code comments do not even try to explain their differences . Names should be clearly different and indicate purposes.
2. Validation function names change in crazy way when passed to child components. In desktop rendering, 'validateTime' is passed as 'validate' and 'validateInputTime' is passed as 'validateTime'. Confusion maximised :)
3. Related to above, there is no reason to pass static code as a prop. Move the validation functions to util and import them from there.